beacon_node/src/cli.rs

```diff
@@ -187,6 +187,12 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .help("One or more comma-delimited trusted peer ids which always have the highest score according to the peer scoring system.")
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("enable-local-discovery")
+                .long("enable-local-discovery")
+                .help("Lighthouse by default does not discover local IP addresses. Set this flag to enable connection attempts to local addresses.")
```

Comment:
These are not really "local" IP addresses.
I'd call them "private" or "intranet" IP addresses instead.